src/coreclr/jit/indirectcalltransformer.cpp

```diff
@@ -758,7 +808,13 @@ class IndirectCallTransformer
             // We know this call can devirtualize or we would not have set up GDV here.
             // So impDevirtualizeCall should succeed in devirtualizing.
             //
-            assert(!call->IsVirtual());
+            if (call->IsVirtual())
```

Comment:
Is this still needed? We can see devirt failures?

Reply:
Ah, it's just a leftover from the initial prototype. not needed now